probe-rs/src/memory/mod.rs

```diff
@@ -238,129 +270,50 @@ pub trait MemoryInterface {
         if address % 4 != 0 || len % 4 != 0 {
             // If we do not support 8 bit transfers we have to bail because we can only do 32 bit word aligned transers.
             if !self.supports_8bit_transfers()? {
-                return Err(Error::MemoryNotAligned {
+                return Err(Into::into(MemoryNotAlignedError {
```

Comment:
Why did you chose `Into::into` over the postfix `.into()`?

Reply:
🤷 why not ? 😄